### PR TITLE
Swedish (sv_SE) translations for SonataAdminBundle translations' domain

### PR DESCRIPTION
**Pull request by tiriana**
I've noticed that Swedish translation is missing from SonataAdminBundle, so I created some basic one.
Hope you'll find it good enough.

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
@tiriana Thanks for your contribution.

I'm wondering: Why sv_**SE** ?

**Comment by Soullivaneuh** [comment]
Ok found why:

```
Swedish - Finland	sv	sv-fi
Swedish - Sweden	sv	sv-se
```

Source: http://www.science.co.il/Language/Locale-codes.asp

But language code seems to be `sv` only.

Do you have a language difference between Finland and Sweden?

Regards.

File: Resources/translations/SonataAdminBundle.sv_SE.xliff [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                <source>link_action_show</source>
+                <target>Visa</target>
+            </trans-unit>
+            <trans-unit id="action_show">
+                <source>action_show</source>
+                <target>Visa</target>
+            </trans-unit>
+        </body>
+    </file>
+</xliff>
```
Comment (Soullivaneuh) [inline Resources/translations/SonataAdminBundle.sv_SE.xliff]: Please add a new line at EOF.

**Comment by tiriana** [comment]
@Soullivaneuh 

about "Do you have a language difference between Finland and Sweden?" - I'm neither Finnish nor Swedish, I'm Polish :) so I don't know all the  differences, but they seem to be vary subtle. I only trust [wikipedia](https://en.wikipedia.org/wiki/Finland_Swedish) about this: 

> (...) there are differences. They are generally in two categories: Old Swedish forms vis-a-vis loanwords and calques from Finnish or independently borrowed from other languages (nowadays mostly English). There are also some terms differing because of differing legislation

I made those translation btw - while doing some project for Swedish customer. I just spent some time with dictionary, and later checking if what I translated makes sense when translating back from Swedish to English. Nevertheless it would be great it some native Swedish speaker would express their opinion about those translations.

About second thing - newline added.

**Comment by Soullivaneuh** [comment]
Let's take this one so. Native Swedish speaker can correct it if something is wrong on it. :+1: 

Can you please squash your commits?

**Comment by tiriana** [comment]
@Soullivaneuh squashed

**Comment by Soullivaneuh** [comment]
Thanks for your contribution. :+1:

**Pull request merged**